packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -273,7 +280,48 @@ export class CXLMarketingNavElement extends LitElement {
         );
 
         // Populate.
-        contextMenu.items = this._createContextMenuItems(menuItem.children);
+        const contextMenuItems = this._createContextMenuItems(menuItem.children);
+        contextMenu.items = contextMenuItems;
+
+        const setDescription = async () => {
+          contextMenu.items = contextMenuItems;
+          contextMenu.render();
+
+          const description = document.createElement('vaadin-context-menu-item');
+          render(html`${unsafeHTML(menuItem.description)}`, description);
+
+          setTimeout(() => {
+            description.style.maxWidth = getComputedStyle(
+              document.querySelector('vaadin-context-menu-list-box')
+            ).width;
+
+            contextMenu.items = [
+              ...contextMenuItems,
+              { component: document.createElement('hr') },
+              { component: description },
+            ];
+            contextMenu.render();
+          }, 0);
+        };
+
+        if (menuItem.description) {
+          contextMenu.addEventListener('opened-changed', () => {
+            if (!document.querySelector('vaadin-context-menu-list-box')) return;
```

Comment:
Wouldn't regular positive check be more sane here?

```js
             if (document.querySelector('vaadin-context-menu-list-box')) {
               setDescription();
             }
```

We're looking to get away from multiple statements on single line, maybe the ESLint standard hasn't been included.

Reply:
Agree